application.lock.json

```diff
@@ -2,13 +2,8 @@
   "name": "app-fqm",
   "version": "1.2.0-SNAPSHOT.100200000001350",
   "description": "Application descriptor for the FOLIO FQM application",
-  "platform": "complete",
```

Comment:
platform should be complete and app-platform-minimal should be listed as a dependency